.env

```diff
@@ -0,0 +1,2 @@
+BROWSER=chrome
+BASE_URL=https://qa-react-conduit.herokuapp.com/
```

Comment:
This .env file should never be pushed to the repository. You can untrack it from git. You can find information about this in this [link](https://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/).